wgengine/magicsock/magicsock.go

```diff
@@ -517,7 +517,7 @@ func appendDests(dsts []*net.UDPAddr, as *AddrSet, b []byte) (_ []*net.UDPAddr,
 			return dsts, roamAddr
 		}
 	}
-	for i := len(as.addrs) - 1; i >= 0; i-- {
+	for i := range as.addrs {
```

Comment:
GitHub sucks and won't let me leave a comment more than 3 lines away from any line you touched (WTAF), so please document up on line 471 the new behavior of appendDst, and what order it uses.

You wrote a huge commit message comment that nobody will ever see again. If it's that subtle, some words belong in the code.